assets/js/homepage.js

```diff
@@ -1,3 +1,24 @@
 $('#itemTable tr').click(function () {
-    $("#modalContainer").load($(this).attr('data-access-url'));
+    $.get($(this).attr('data-access-url'), (data) => {
+        // we can't just load the new content into the page with innerhtml because that won't execute scripts
+        // we can't use jquery load because that would inline scripts breaking CSP
+        // NOTE: this only working for scripts loaded with an src attribute and ignores all other attributes
+        const div = document.createElement("div");
+        div.innerHTML = data;
+        // document.getElementById("modalContainer").appendChild(div); // insert the content into the page scripts don't run
```

Comment:
Remove this line

Reply:
I tried removing this and the modal would not load any longer.

Reply:
It broke when you removed the comment?

Reply:
Oh, no not the comment. I wasn't actually sure which line you were referring to so i assumed it was all of them. Did you mean the commented-out code?

Reply:
Yeah, the comment isn't needed